src/styles/blocks/card.scss

```diff
@@ -0,0 +1,48 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  grid-column: span 2;
+
+  @include on-tablet {
+    grid-column: span 6;
+  }
+
+  @include on-desktop {
+    grid-column: span 4;
+  }
+
+  &__image {
+    display: block;
+    width: 100%;
+
+    margin-bottom: 24px;
+  }
+
+  &__product-name {
+    font-family: 'Space Grotesk', serif;
```

Comment:
And use it in all cases. 
Try to think, maybe the best way use font inheriting ( not about CSS property)